dotcom-rendering/src/components/Discussion.tsx

```diff
@@ -91,22 +92,41 @@ export const Discussion = ({
 	user,
 	idApiUrl,
 }: Props) => {
-	const [commentPage, setCommentPage] = useState<number>(1);
+	const [commentPage, setCommentPage] = useState(1);
 	const [commentPageSize, setCommentPageSize] = useState<25 | 50 | 100>();
 	const [commentOrderBy, setCommentOrderBy] = useState<
 		'newest' | 'oldest' | 'recommendations'
 	>();
-	const [isExpanded, setIsExpanded] = useState<boolean>(false);
+	const [commentCount, setCommentCount] = useState(0);
```

Comment:
We have `useCommentCount` for this specific purpose

Reply:
Ah, great, thanks Max! I didn't know that existed. I'll update the code to use that 👍

Reply:
I’ve actually discovered that `commentCount` and `topLevelCommentCount` cannot be conflated… the former relates to the number of comments, the latter to the number of threads!

Fix in #10477